lib/ansible_tower_client/base_model.rb

```diff
@@ -78,7 +91,7 @@ def key_to_attribute(key)
       end
 
       def generate_writer?
-        false
+        true
```

Comment:
What if I added a `save` method so in the case a user does intend to modify the in memory object, they have a way to persist their changes in an AR fashion?